serverless/lambda/src/main/java/uk/gov/di/services/ValidationService.java

```diff
@@ -32,4 +34,21 @@
         }
         return Optional.empty();
     }
+
+    public Optional<ErrorResponse> validatePhoneNumber(String phoneNumberInput) {
+        phoneNumberInput = phoneNumberInput.replaceAll("\\s+", "");
+        if ((!phoneNumberInput.matches("[0-9]+")) || (phoneNumberInput.length() < 10)) {
```

Comment:
<= 10 ?

Reply:
Can't some UK numbers be 10? Tracy mentioned that on the content doc